.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
We'll have to extend this, but fine for now.

Reply:
I'm not sure this should be restricted to PRs to `master`. Visual regression might since that's more of a linear approval to a single branch.